### PR TITLE
Darkwing Daphne's Typo Thing

### PR DESCRIPTION
**Pull request by daphnegold**
🐞🐛🐜🐝

### CODE REVIEW

**Comment by kariabancroft** [comment]
You should probably not have the DB files in here, I would delete them as well as add them to the gitignore

File: features/create_or_edit_category.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Feature: Write Articles
+  As a blog administrator
+  In order to organize my amazing thoughts
+  I want to be able to categorize articles
+
+  Background:
+    Given the blog is set up
+    And I am logged into the admin panel
+
+  Scenario: Visit the category page
```
Comment (kariabancroft) [inline features/create_or_edit_category.feature:11]: Does this test if the category is saved or edited successfully?

File: spec/controllers/admin/categories_controller_spec.rb [hunk 2]
```diff
@@ -16,18 +16,27 @@
     assert_response :redirect, :action => 'index'
   end
 
-  describe "test_edit" do
-    before(:each) do
-      get :edit, :id => Factory(:category).id
+  describe "test_new_or_edit" do
+    # before(:each) do
```
Comment (kariabancroft) [inline spec/controllers/admin/categories_controller_spec.rb]: If you don't want this commented out portion, it should be removed

File: features/create_or_edit_category.feature (hunk 1 above)
Comment (daphnegold) [inline features/create_or_edit_category.feature:11]: It didn't, but now it does!

File: spec/controllers/admin/categories_controller_spec.rb (hunk 2 above)
Comment (daphnegold) [inline spec/controllers/admin/categories_controller_spec.rb]: Done

**Comment by kariabancroft** [comment]
Sorry about the late breaking comment - but your db schema file should also probably not be in your commits either

**Comment by daphnegold** [comment]
@kariabancroft I think I finally got it right this time with the `db/*.sqlite*` and `db/schema.rb` files removed like the original state of the repo.

**Pull request closed without merging**